modules/cnv_master/1.0/src/R/merge_segs.R

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env Rscript
+#
+
+
+suppressWarnings(suppressPackageStartupMessages({
+
+  message("Loading packages...")
+
+  required_packages <- c(
+    "readr", "dplyr", "tidyr", "purrr"
+  )
+
+  installed_packages <- rownames(installed.packages())
```

Comment:
I think installing packages in a Snakemake-run script causes problems especially if more than one instance of this is running simultaneously. Doesn't the conda env take care of installing the packages?

Reply:
You are right, these packages are already there from conda env. I will just load them here and skip the installation part

Reply:
> Thanks Laura! My plan to handle unix groups in GAMBL is to have unix_group-specific outputs that would be merged then to `all_the_things`. I am thinking of this way because it will be then straightforward to set GAMBLR config for permission-defined access to flat files with merges. This can be converted to a GAMBL-specific snakefile if you think this will be a better approach. I thought of having it as a module because we would need a merged seg file as input to lymphgen module in upcoming lymphgen-centric project snakefile as part of demo

This makes sense, although the alternative we've discussed is having LymphGen run per-sample rather than on the merge. That kind of update would still have to leverage the input functions you've used here to get the best CNV output though, and doesn't negate the need for individual unix_group merges in GAMBL. 

From a practicality standpoint and to avoid having to wait for merges before re-running LymphGen I favour the per-sample approach, especially now that the merges we're generating will be a perfect concatenation of the input maf files.

Reply:
I don't know why GitHub put my quote reply in this conversation...

Reply:
I see, right - we can run lymphgen per sample. For capture/exome data, the SVs from FISH would be supplied in a separate file and I guess it would not really affect the module if FISH data is per-sample or all in one file, so we can still run it individually

Reply:
Awesome. This module is still useful though!

Reply:
This has been addressed 👍

Reply:
Another thing I also just remembered is that initially, I thought of this module to prepare inputs for GISTIC and SV-hotspot. The SV-hotspot did not prove to be that useful, but I am still planning on the GISTIC module and this merging would be needed there

Reply:
Excellent. Another good reason to keep this.